vochain/scrutinizer/queries/vote_reference.sql

```diff
@@ -0,0 +1,13 @@
+-- name: CreateVoteReference :execresult
+INSERT INTO vote_references (
+	nullifier, process_id, height, weight,
+	tx_index, creation_time
+) VALUES (
+	?, ?, ?, ?,
+	?, ?
+);
+
+-- name: GetVoteReference :one
+SELECT * FROM vote_references
```

Comment:
Just a question: vote_references contains all vote references of all processes or all vote references for a given process? I understand it contains all vote references of all processes.
If that is the case could this query be faster filtering first by processId ?

Thank you

Reply:
the tables are all global, so it contains entries for all processes.

I haven't spent any time optimizing the queries or tables, beyond adding column indexes where it makes sense for sure :) it's entirely likely that we'll want indexes on vote_references by process ID as well, but I'd rather do that in the future when we can prove with some benchmarks that it actually helps.